main/dns-root-hints/APKBUILD

```diff
@@ -1,39 +1,69 @@
 # Contributor: Leonardo Arena <rnalrd@alpinelinux.org>
 # Contributor: tcely <dnshints-root+aports@tcely.33mail.com>
 # Maintainer: tcely <dnshints-root+aports@tcely.33mail.com>
-pkgname=dns-root-hints
-pkgver=2019031302
-pkgrel=1
+pkgname="dns-root-hints"
+pkgver="2019073000"
+pkgrel=0
 pkgdesc="The DNS root hint(s)"
 url="https://www.internic.net/domain"
 arch="noarch"
 license="Public-Domain"
 makedepends="curl gnupg"
 options="net"
-source="verisign-grs-nstld-key.asc named.root named.root.sig"
-builddir="$srcdir/$pkgname-$pkgver"
+source="
+	${pkgname}-${pkgver}.tar.gz::https://github.com/tcely/internic-domain-data/archive/${pkgver}.tar.gz
+	verisign-grs-nstld-key.asc
+	"
+builddir="${srcdir}/internic-domain-data-${pkgver}"
 
-build() {
-	mkdir -p "$builddir"
+prepare() {
+	default_prepare
+
+	builddir="${builddir}/build"
+	mkdir -p "${builddir}"
 }
```

Comment:
Although it's a nitpick don't we usually set builddir to $srcdir/<name>/build already and just cd to $srcdir/<name> in prepare?

Reply:
Normally, `builddir` is set to the directory that is contained in the source archive. This is usually `$pkgname-$pkgver` that is created by the unpack function under `$srcdir` which is why that is the default.

This package was created with an empty build directory in mind. So when extracting the source archive, the easiest way back to that state is to create an empty directory.